frontend/src/views/Game.tsx

```diff
@@ -158,22 +179,26 @@ function GamePage() {
 
     // Get the result of promises and set the default code list.
     Promise.all(promises).then((result) => {
-      const codeMap = result[0];
+      const codeList = [];
+
+      for (let i = 0; i < result.length; i += 1) {
+        codeList.push(result[i][language[i]]);
```

Comment:
how does this work? it indexes language[i] but it seems like language is always a list of length 1 with either the default Python or whatever language the last submission was in?

Reply:
I think this was updated on the latest PR #152, so I'll resolve this comment.